imgui.ini

```diff
@@ -0,0 +1,45 @@
+[Debug]
+Pos=60,60
+Size=400,400
+Collapsed=0
+
+[Splash Screen Banner]
+Pos=522,448
+Size=1344,756
+Collapsed=0
+
+[nostreaming_popup]
+Pos=50,100
+Size=250,50
+Collapsed=0
+
+[Firmware update recommended##1]
+Pos=1014,55
+Size=315,110
+Collapsed=0
+
+[Notification parent window]
+Pos=60,60
+Size=32,32
+Collapsed=0
+
+[Stream of 1]
+Pos=343,53
+Size=996,32
+Collapsed=0
+
+[Footer for stream of 1]
+Pos=343,608
+Size=120,32
+Collapsed=0
+
+[header of 3dviewer]
+Pos=340,50
+Size=1004,32
+Collapsed=0
+
+[3D Info box]
+Pos=1079,87
+Size=260,65
+Collapsed=0
```

Comment:
What's this for?

Reply:
Not sure where this file came from, I'll remove it and make sure it doesn't break anything.